README.md

```diff
@@ -0,0 +1,131 @@
+<h3><b>School Library</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 School Library <a name="about-project"></a>
+
+School library is a backend project with ruby .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+<ul><li>Ruby</li></ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Student class**
+- **Person class**
+- **teacher class**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- **Live demo not available at this time**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Ruby
+
+### Setup
+
+Clone the repo, and install the dependencies
+
+### Install
+
+Install this project with:
+
+```sh
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> I am the sole author of this project.
+
+👤 **Grace Htet**
+
+- GitHub: [Grace Htet](https://github.com/GraceHtet)
+- LinkedIn: [Grace Htet](https://linkedin.com/in/thirihtethtetaung)
+- Twitter: [Grace Htet](https://twitter.com/Grace_Htet4)
+- Angelist: [Grace Htet](https://wellfound.com/u/thiri-htet)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Decorate class**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project or find it useful, please consider giving it a ⭐️. Thanks!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+My coding partner and Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [OPTIONAL]
Let's improve ReadMe the look of the ReadMe by removing the underlined part in the screenshot below:

![image](https://user-images.githubusercontent.com/12745474/232433444-6a91be6d-331d-4f03-87e1-024d1117238e.png)